lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -43,6 +43,7 @@ def []=(str, value)
 end
 
 RSpec.configure do |config|
+  config.include LogStashHelper
```

Comment:
both extend and include are required for now because of the "legacy" dsl addons.